crates/ruma-common/CHANGELOG.md

```diff
@@ -14,7 +14,17 @@ Breaking changes:
   query parameters. Note that the (de)serialization of the type used must work
   with `serde_html_form`.
 - The `header` attribute for the `request` and `response` macros accepts any
-  type that implements `ToString` and `FromStr`. 
+  type that implements `ToString` and `FromStr`.
+- The `compat-key-id` cargo feature was renamed to
+  `compat-server-signing-key-version`.
+- `(Owned)KeyName` was renamed to `(Owned)ServerSigningKeyVersion` and is now
+  validated according to the set in the docs, unless the
```

Comment:
I guess this is a set of crypto algorithms? May be good to expand this entry a bit to make it clearer.

Reply:
No a set of allowed characters in the string.